### DISCUSSION TITLE
[help] Run CLI on Chia GUI

### DISCUSSION THREAD

**Discussion by KeramxD** [Chia-Network/chia-blockchain#3343] [Support]
Hello,
I have installed Chia GUI via dpkg. Where it installs Chia? How can I run Chia CLI?

**Comment by lvcivs** [comment]
It installs to `/usr/lib/chia-blockchain`

The executable is at `/usr/bin/chia-blockchain`

**Comment by KeramxD** [comment]
Can I run CLI somehow? When I try `. ./activate` in these directories it shows `-bash: ./activate: No such file or directory`

**Comment by sargonas** [comment]
I could be wrong in my memory, but I believe when you install in that method, I don't think the python venv method is needed or included.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
